pool/app/query/entity.ml

```diff
@@ -151,11 +167,33 @@ type t =
   }
 [@@deriving eq, show]
 
+let to_query_string { pagination; search; sort } =
+  [ pagination |> Option.map Pagination.to_query_parts
+  ; search |> Option.map Search.to_query_parts
+  ; sort |> Option.map Sort.to_query_parts
+  ]
+  |> List.map (Option.value ~default:[])
+  |> List.flatten
+  |> List.map (fun (k, v) -> String.concat "=" [ k; Uri.pct_encode v ])
+  |> String.concat "&"
+;;
+
+let pp fmt t = Format.fprintf fmt "%s" (to_query_string t)
```

Comment:
**note**. in this case, since the query printing is complex, we don't want to make assumptions about where it will be printed to (like you'd do with `to_string`), so a `pp` function is more appropriate. Later on in this PR we use this to compose the final URL for table updates.